pkg/release/build_version.go

```diff
@@ -46,6 +46,8 @@ const (
 )
 
 var (
+	BranchRegex          = regexp.MustCompile(git.BranchRE)
```

Comment:
nit: feels like `BranchRE` might be more appropriate to be exported from `release` and used in `git`

Reply:
Yes, I'll fix that by putting the regex'es into a new package. This has to be done because otherwise we would run into an import cycle.